technology/java-spring-boot/pom.xml

```diff
@@ -154,8 +149,53 @@
       <plugin>
         <groupId>org.graalvm.buildtools</groupId>
         <artifactId>native-maven-plugin</artifactId>
+        <configuration>
+          <imageName>app.native</imageName>
+          <buildArgs>
+            <!-- Ob == fast build -->
+            <buildArg>-Ob</buildArg>
+            <buildArg>--no-fallback</buildArg>
+            <buildArg>--trace-class-initialization=org.junit.platform.engine.TestTag</buildArg>
+          </buildArgs>
+        </configuration>
       </plugin>
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${version.surefire.plugin}</version>
+            <configuration>
+              <excludedGroups>slowly</excludedGroups>
```

Comment:
By using the following, you'll be able to remove the entire `default` profile:

https://junit.org/junit5/docs/5.1.0/api/org/junit/jupiter/api/condition/EnabledIfSystemProperty.html

Reply:
Nice catch!